### ISSUE TITLE
Crash on infer -- ./gradlew

### ISSUE THREAD

**Issue by FreddieMigme** [facebook/infer#97]
Got a crash in 0.11
```
Fatal error: exception File "java/jTransStaticField.ml", line 198, characters 14-20: Assertion failed
```

**Comment by sblackshear** [comment]
Thanks for reporting. Out of curiosity, do you have a small example that reproduces the issue? I think I know how to fix the problem, but would help to have a test.

**Comment by sblackshear** [comment]
This should be fixed by 4feb925dd7cab80178de283a17970caf31cd6f6c; feel free to re-open if you see the issue again.
